nemo/collections/asr/models/hybrid_rnnt_ctc_bpe_models.py

```diff
@@ -94,7 +95,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
         )
 
         # Setup wer object
-        self.wer = RNNTBPEWER(
+        self._wer = WER(
             decoding=self.decoding,
```

Comment:
Don't assign to _wer - always assign to self.wer - let setter property handle it.